src/app/containers/StoryPromo/LinkContents/index.test.jsx

```diff
@@ -34,6 +34,16 @@ const mediaItem = {
   },
 };
 
+const photogalleryItem = {
```

Comment:
For some reason this file doesn't contain what has been merged when adding the PGL to the whitelist: 
https://github.com/bbc/simorgh/blob/ff80eea8842dae7ec864538443c330600cbc589f/src/app/containers/StoryPromo/MediaIndicator/index.test.jsx

Reply:
I'm a bit confused about where the difference is. I have updated the branch though - is the issue still there?

Reply:
My bad. Confused this file with `MediaIndicator/index.test.jsx`